ckanext/harvester_data_qld_geoscience/geoscience_dataset.json

```diff
@@ -272,5 +272,6 @@
       "display_snippet": null,
       "validators": "ignore_missing unicode_safe"
     }
-  ]
-}
\ No newline at end of file
+  ],
+  "resource_fields": []
```

Comment:
So this is the only 'change' for dataset routing?

The rest of the PR is for alignment to our new standard CICD pipeline. Is that correct?

Reply:
Yes.

It's a necessary change, though, otherwise we get HTTP 500 on Geoscience datasets.